src/db/api/beacon/beacon.ts

```diff
@@ -232,8 +234,9 @@ export class BeaconDB extends DatabaseService implements IBeaconDb {
   }
 
   public async deleteGenesisDeposits(deposits: Deposit[]): Promise<void> {
-    const criteria: any[] = deposits.map((deposit) => {
-      encodeKey(Bucket.genesisDeposit, deposit.index);
+    const criteria: (Buffer | string)[] = [];
+    deposits.map((deposit) => {
+      criteria.push(encodeKey(Bucket.genesisDeposit, deposit.index));
```

Comment:
It looks like the issue was the `map`ed function was not `return`ing the encoded key.

I think we would still want the implementation we had before, but with the `return`.
Or change the `map` to a `forEach`.
But I think the map is, subjectively, more readable.

Reply:
updated